### ISSUE TITLE
gpu: cuda: benchdnn conv fails with Assertion

### ISSUE THREAD

**Issue by hdelan** [oneapi-src/oneDNN#1275]
# Summary

benchdnn --conv fails with 
```
benchdnn: /home/hughdelaney/oneDNN/tests/benchdnn/conv/conv.cpp:648: void conv::check_known_skipped_case(const conv::prb_t *, res_t *): Assertion `!"unknown post-op type"' failed.
```
For Nvidia Titan, Nvidia GTX1050. Error not seen on A100.

# Version 
oneDNN f805f12d9c4f268307bf3277bafcf51e769abbd1
intel/llvm c690ac8d771e8bb1a1be651872b782f4044d936c
OS tested: Ubuntu 18.04, Arch Linux
GPU: Nvidia Titan, Nvidia GTX1050

# To reproduce:
```
 ./benchdnn --conv --engine=gpu --batch=inputs/conv/test_conv_ci
```

**Comment by dzarukin** [comment]
Hi @hdelan, thanks for catching this. Sources don't check for prelu post-op. Will fix it soon.

**Comment by dzarukin** [comment]
Should be fixed by [4353a78](https://github.com/oneapi-src/oneDNN/commit/4353a785fae3f9aa1f106a8e586a2931882cc317). Thank you.

**Comment by hdelan** [comment]
Great thanks
